src/objectmanager.js

```diff
@@ -101,12 +104,43 @@ hbbtv.objectManager = (function() {
         }
     }
 
+    function updateImageSrc(imageElement) {
```

Comment:
This part should be modified.
I just found the browser selects the appropriate image based on the device's pixel density using the srcset attribute and the x descriptor.
window.devicePixelRatio = 2 as my mitxpert test is running on 1080p emulator
window.devicePixelRatio = 1.33 as running on 720p emulator

For this case, 
  <img id="cimg" src="low.png" srcset="low.png 1x, medium.png 1.5x, high.png 3x " />
the browser will select medium.png as running either 1080p or 720p emulator.

However,
I reckon, according to spec, the terminal should make the browser select low.png for 720p emulator and medium.png for 1080p emulator.